Makefile

```diff
@@ -86,6 +87,14 @@ image:
 	Dockerfile \
 	.
 
+docker-ext:
+	$(DOCKER_CMD) buildx build \
+	--platform=linux/amd64,linux/arm64 \
+	--push \
+	-t $(DOCKER_EXT_REPO)/$(DOCKER_IMAGE_NAME)-dd-extension:$(DOCKER_IMAGE_VERSION) -f \
```

Comment:
this will tag the docker image as "headlamp-dd-extension". Up for better naming suggestions.

Reply:
Seems fine to me. I guess headlamp-docker-desktop-extension was too long?

Reply:
>  I guess headlamp-docker-desktop-extension was too long

yup